app/js/validateQuestionsFormData.js

```diff
@@ -0,0 +1,180 @@
+let allAnswers = document.querySelectorAll('.answer')
+
+allAnswers.forEach(function(answer, index) {
+    answer.addEventListener('input', () => {
+        event.target.nextElementSibling.innerHTML = "<input type=\"radio\" name=\"answer_radio_button\" value=" + (index + 1) +  "><p>Correct Answer</p>"
+    })
+})
+
+allAnswers.forEach(function (answer) {
+    answer.addEventListener('blur', () => {
+        if (!event.target.value.trim()) {
+            event.target.nextElementSibling.innerHTML = ''
+        }
+    })
+})
+
+document.querySelector('.add_question').addEventListener('submit', function(event) {
+    event.preventDefault()
+
+    let newQuestion = {}
+    let question = document.getElementById('question').value
+    let answer1 = allAnswers[0].value
```

Comment:
this is a questionable way of doing this, but i will let it go. in real life i would make you do 1 selector for each input